src/Controls/src/Core/Element/Element.cs

```diff
@@ -43,7 +43,9 @@ public abstract partial class Element : BindableObject, IElementDefinition, INam
 
 		IList<Element> _internalChildren;
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Element.xml" path="//Member[@MemberName='AutomationId']/Docs/*" />
+		/// <summary>Gets or sets a value that allows the automation framework to find and interact with this element.</summary>
+		/// <value>A value that the automation framework can use to find and interact with this element.</value>
```

Comment:
Personally I don’t think I used the Value tag too much. Is there any specific reason you do? Does it give us any extras?

Reply:
Personally, I didn't find super useful. _**But**_, it was being used in the old xml docs, so I figured I might as well copy it over, and the API docs guide implies you should use it: https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value.

It also reflects on MSLearn  adding a section "Property Value". Happy to remove the tag if you think that's the best way to go about it

Reply:
My view: I think it's important to have the <value> entry. It's just that I don't think the current text is that useful - it essentially repeats the text from the summary.